test/unit/model/LtiDeliveryFactoryTest.php

```diff
@@ -15,11 +15,13 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2019 (original work) Open Assessment Technologies SA;
+ * Copyright (c) 2019-2020 (original work) Open Assessment Technologies SA;
```

Comment:
Why we updated this class? Is it missing some test code? No test code was added / changed.

Reply:
not inside the scope of the bugfix

Reply:
i think the intent was to do a cleanup